lib/myfinance/resources/payable_account.rb

```diff
@@ -14,12 +14,19 @@ def pay(id, entity_id, params = {})
         end
       end
 
+      def undo_payment(id, entity_id)
+        http.put(endpoints(id, entity_id, :undo_payment)) do |response|
+          respond_with_object(response, 'payable_account')
+        end
+      end
+
       private
 
       def endpoints(id, entity_id, key)
         {
           create: '/entities/:entity_id/payable_accounts',
-          pay: '/entities/:entity_id/payable_accounts/:id/pay'
+          pay: '/entities/:entity_id/payable_accounts/:id/pay',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.